backend/corpora/common/entities/entity.py

```diff
@@ -62,3 +63,45 @@ def __getattr__(self, name):
         :return:
         """
         return self.db_object.__getattribute__(name)
+
+    @classmethod
+    def _create_sub_objects(
+        cls, rows: typing.List[dict], db_table: Base, add_columns: dict = None, primary_keys: typing.List[str] = None
+    ) -> typing.List[Base]:
+        """
+        Create or modify N `rows` in `db_table` associated with Entity Object during object creation. If id is provided,
+        then the row with that id is retrieved and updated, else a new UUID is generated and a new row is created.
+
+        :param rows: A list of dictionaries each specifying a row to insert or modify
+        :param db_table: The Table to add or modify rows
+        :param primary_keys: Additional columns required to build the primary key. This is used when the primary consist
+        of
```

Comment:
Weird line break here and line 79. I think they can be combined.

Reply:
fixed